paywall/src/__tests__/unlock.js/CheckoutUIHandler.test.ts

```diff
@@ -74,7 +77,7 @@ describe('CheckoutUIHandler', () => {
     function makeReadyCheckout() {
       fakeWindow = new FakeWindow()
       const handler = makeCheckoutUIHandler(fakeWindow)
-      handler.init()
+      handler.init(false)
```

Comment:
nit: do you mind adding a comment to indicate what `false` is for, like `handler.init(false /* usingManagedAccount */)`

Reply:
As an alternative, what about passing it in as an object? That would enforce that it's always clear at the caller what that value is